app/controllers/concerns/site_config_params.rb

```diff
@@ -0,0 +1,16 @@
+module SiteConfigParams
```

Comment:
**suggestion (non-blocking):** I'm personally not a big of concerns like this one. A service like `SiteConfigs::PrepareParams.call(params)` is a lot more explicit and doesn't require us to `include` a module in the controllers.